userguide/intro.rst

```diff
@@ -247,8 +247,8 @@ These screen options have changed:
 * The :guilabel:`Hold Pending Snapshots` checkbox has been added to
   :menuselection:`Tasks --> Replication Tasks --> ADD`.
 
-* Replication Task log files can be displayed and downloaded in
-  :menuselection:`Tasks --> Replication Tasks`.
+* Replication Task log files are displayed and downloaded in
```

Comment:
Displaying or downloading log files is an optional action, so "can be" was correct here.